elixir/apps/domain/lib/domain/auth.ex

```diff
@@ -398,8 +515,15 @@ defmodule Domain.Auth do
   end
 
   defp build_subject_expires_at(%Actors.Actor{} = actor, expires_at) do
+    now = DateTime.utc_now()
+
     default_session_duration_hours = Map.fetch!(@default_session_duration_hours, actor.type)
-    expires_at || DateTime.utc_now() |> DateTime.add(default_session_duration_hours, :hour)
+    expires_at = expires_at || DateTime.add(now, default_session_duration_hours, :hour)
+
+    max_session_duration_hours = Map.fetch!(@max_session_duration_hours, actor.type)
```

Comment:
TODO: increase limits and split tokens to allow non-admin-ui tokens to be long lasting for admins

Reply:
Moved to a separate issue